core/chains/evm/client/config_builder.go

```diff
@@ -76,18 +85,37 @@ func NewClientConfigs(
 	return chainConfig, nodePoolCfg, nodes, nil
 }
 
+// verifyLogBroadcasterFlag checks node config and return error if LogBroadcaster enabled but some node missing WSURL
+func verifyLogBroadcasterFlag(nodes []*toml.Node, LogBroadcasterEnabled bool) error {
```

Comment:
Config Builder is built for external applications like Atlas, where LogBroadcaster does not exist. The CL Node does not use it. 
`verifyLogBroadcasterFlag` should be moved to CL Node config validation, which is located in [toml pkg](https://github.com/smartcontractkit/chainlink/blob/1ea9f79793f646977b44e38a34b2e70c28b2849e/core/chains/evm/config/toml/config.go#L291)

Reply:
Do we need to check the `LogBrodcasterEnabled` flag for node during [validateConfig()](https://github.com/smartcontractkit/chainlink/blob/1ea9f79793f646977b44e38a34b2e70c28b2849e/core/chains/evm/config/toml/config.go#L971) ?

Reply:
Oh okay, so we don't care about the node config validation [here](https://github.com/smartcontractkit/chainlink/blob/1ea9f79793f646977b44e38a34b2e70c28b2849e/core/chains/evm/config/toml/config.go#L971) because it's for external users ?